firecracker-control/local.go

```diff
@@ -306,6 +325,7 @@ func (s *local) getVM(id string) (*instance, error) {
 	s.vmLock.RUnlock()
 
 	if !ok {
+		log.L.Warnf("attempt to get not existing VM id: %s", id)
```

Comment:
We call getVM to check if the VM exists when creating a new one, so will we see this log every time that happens? If so, I don't think a WARN log should be written for something that's expected to happen. Maybe letting the caller decide whether to log would be more flexible in this case.